app/styles/_base.scss

```diff
@@ -1,3 +1,5 @@
+@import 'vars';
+
```

Comment:
Remove: https://github.com/osdc/hermes/pull/3/files#diff-da132c0d50c9cd8406b6391530936e2bR7 from inside `body` and use it in `.container` instead. Since it's currently leading to the content in header and footer being misaligned. For example, look at "Made with ❤ by OSDC" below: 
<img width="692" alt="screen shot 2016-12-31 at 2 32 58 pm" src="https://cloud.githubusercontent.com/assets/7583815/21576846/1a56a01c-cf66-11e6-9c13-03bbd3e5bf45.png">